packages/parallax/src/index.tsx

```diff
@@ -90,6 +110,23 @@ export const ParallaxLayer = React.memo(
 
       React.useImperativeHandle(ref, () => layer)
 
+      const layerRef = useRef<any>()
+
+      const setSticky = (height: number, scrollTop: number) => {
+        const from = sticky!.from * height
```

Comment:
This is a similar thing to above, we should look at how `Controller` handles a lack of `from` or `to`